core/src/main/java/hudson/model/Queue.java

```diff
@@ -1809,6 +1834,10 @@ public WaitingItem(Calendar timestamp, Task project, List<Action> actions) {
             this.timestamp = timestamp;
         }
 
+        public static int getCurrentCounterValue() {
+            return COUNTER.intValue();
+        }
```

Comment:
Simply so it can be accessed and checked from a test. Could make it `protected`.